modules/component/src/index.ts

```diff
@@ -1,11 +1,8 @@
-export { Hook$ } from './hook$/hook$.decorator';
```

Comment:
Inside the `hooks.decorator` file, we have to change the import from `ɵNG_COMPONENT_DEF` to `ɵNG_COMP_DEF`, as this has been renamed.

Reply:
This was renamed in v9. It's marked as private so it can be renamed in even a minor or patch release of Angular. It's risky to depend on unstable APIs.

Reply:
There is no hooks decorator!

Reply:
Something went wrong with this merge!!

Reply:
Seems like the hooks decorator was already added to the ngrx branch by Brandon.
I will push the change to get the build resolved so it isn't blocking, we'll ask Brandon what the plans are with the hooks decorator.